### PR TITLE
chore: [WT-2088] Remove textconfig translations

### PR DESCRIPTION
**Pull request by dreamoftrees**
# Summary
Refactored out existing use of textConfig as it's obsolete and been replaced internally with i18next.
https://immutable.atlassian.net/browse/WT-2088

### CODE REVIEW

**Review by sharonsheah — DISMISSED** [review]
Nice work!

File: packages/checkout/widgets-lib/src/components/NotEnoughGas/NotEnoughGas.tsx [hunk 1]
```diff
@@ -35,14 +35,16 @@ export function NotEnoughGas({
   onAddCoinsClick,
 }:
 NotEnoughGasProps) {
-  const { content, buttons } = text.drawers.notEnoughGas;
-
+  // const { content, buttons } = text.drawers.notEnoughGas;
```
Comment (imx-mikhala) [inline packages/checkout/widgets-lib/src/components/NotEnoughGas/NotEnoughGas.tsx:38]: nit: remove comment

Comment (dreamoftrees) [reply packages/checkout/widgets-lib/src/components/NotEnoughGas/NotEnoughGas.tsx]: Nice catch!

**Review by sharonsheah — APPROVED** [review]

File: packages/checkout/widgets-lib/src/widgets/swap/helpers.ts [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import { TokenInfo, IMTBLWidgetEvents } from '@imtbl/checkout-sdk';
-import { text } from '../../resources/text/textConfig';
-import { SharedViews } from '../../context/view-context/ViewContext';
+import { useTranslation } from 'react-i18next';
```
Comment (imx-mikhala) [inline packages/checkout/widgets-lib/src/widgets/swap/helpers.ts:2]: will need to change how this is handled in the ts files as cant use hooks outside of react components

**Review by sharonsheah — APPROVED** [review]

**Review by sharonsheah — APPROVED** [review]

**Review by imx-mikhala — APPROVED** [review]

**Pull request merged**